.github/workflows/run_tests.yml

```diff
@@ -99,9 +99,9 @@ jobs:
       - name: Run tests
         run: go test -v -race ./...
 
-  tests_win:
-    name: Windows, Go (1.17)
-    runs-on: windows-2019
+  tests_wsc:
+    name: Windows Server Core, Go (1.17)
```

Comment:
Doesn't make much sense specifying 1.17 here, it'll get out of date quickly (or will need to be replaced with every Go update).

Reply:
Maybe not a big deal though, as it's changed by the subsequent commits anyway.

Reply:
It was made for naming unification, because Ubuntu-based tests had matrix strategy with proper go version displayed.